packages/flutter/lib/src/widgets/form.dart

```diff
@@ -141,6 +140,15 @@ class Form extends StatefulWidget {
   /// {@macro flutter.widgets.form.autovalidateMode}
   final AutovalidateMode autovalidateMode;
 
+  /// Used to enable/disable form fields auto validation and update their error
+  /// text.
+  @Deprecated(
+    'Use autoValidateMode parameter which provide more specific '
+    'behaviour related to auto validation. '
+    'This feature was deprecated after v1.19.0.'
+  )
+  final bool autovalidate;
```

Comment:
Does this need to be saved on the class instance, or could it just be a parameter since it's only used in the constructor?

Reply:
I'm adding it back into the class instance because looks like it is being used as by developers, please see https://github.com/flutter/flutter/pull/61648/files#r490653719.

Reply:
Oh got it!